modules/RouterContext.js

```diff
@@ -1,5 +1,6 @@
 import invariant from 'invariant'
 import React from 'react'
+import * as ReactIs from 'react-is'
```

Comment:
Can you import just `isValidElementType` so there is potentially some tree shaking here?

Reply:
Sure, thanks 👍